### ISSUE TITLE
How to train the model?

### ISSUE THREAD

**Issue by framsc** [yluo42/TAC#4]
Could you please share the training script? I can not reproduce the results after reading
FaSNet, https://doi.org/10.1109/ASRU46091.2019.9003849
TAC, https://doi.org/10.1109/ICASSP40776.2020.9054177

Thanks in advance.

**Comment by yluo42** [comment]
Sorry for my late reply as I missed this issue. I will probably prepare one script for training by early Nov.

On the other hand, some recent literatures have used the FaSNet+TAC model as baselines (e.g. https://arxiv.org/abs/2010.01703 and https://arxiv.org/abs/2010.06007), so the models should be able to be reproduced.

**Comment by framsc** [comment]
Thanks for the reply. I have read these papers too. It seems that the researchers are quite interested in FaSNet and TAC. Maybe you can provide a pre-trained model thus we have the same model to talk about.  So are the best hyper-parameters, such as batch size, learning rate. 

Nice job. Hope to hear from you soon.

**Comment by ghost** [comment]
How to train the fasnet model after getting the dataset generated.
